.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,101 @@
+---
+name: "🐞 Bug report"
+description: Report a bug
+title: "(Short description)"
+labels: [bug]
+assignees: [NelsonBN]
+
+
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: What is the problem? A clear and concise description of the bug.
+    validations:
+      required: true
+
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      description: |
+        What did you expect to happen?
+    validations:
+      required: true
+
+
+  - type: textarea
+    id: current
+    attributes:
+      label: Current behavior
+      description: |
+        What actually happened?
+        
+        Please include full errors, uncaught exceptions, stack traces, and relevant logs.
+        If service responses are relevant, please include wire logs.
+    validations:
+      required: false
+
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction Steps
+      description: |
+        Provide a self-contained, concise snippet of code that can be used to reproduce the issue.
+        For more complex issues provide a repo with the smallest sample that reproduces the bug.
+        
+        Avoid including business logic or unrelated code, it makes diagnosis more difficult.
+        The code sample should be an SSCCE. See http://sscce.org/ for details. In short, please provide a code sample that we can copy/paste, run and reproduce.
+    validations:
+      required: false
+
+      
+  - type: textarea
+    id: solution
+    attributes:
+      label: Possible Solution
+      description: |
+        Suggest a fix/reason for the bug
+    validations:
+      required: false
+
+
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional Information/Context
+      description: |
```

Comment:
you recon, instead of description, we could call this a `Notes` field?
I know we could have both, but that would just be redundant...